### PR TITLE
Update Language.de.xml

### PR DESCRIPTION
**Pull request by Mr-Update**
- Correct typos
- Translation improved

### CODE REVIEW

**Comment by idrassi** [comment]
Thank you for this contribution.
@Atalanttore, @andreas-becker : any comments on the changes?

File: Translations/Language.de.xml [hunk 1]
```diff
@@ -178,23 +178,23 @@
     <entry lang="de" key="IDC_RESET_HOTKEYS">Zu&amp;rücksetzen</entry>
     <entry lang="de" key="IDC_SELECT_DEVICE">Datent&amp;räger ...</entry>
     <entry lang="de" key="IDC_SELECT_FILE">&amp;Datei ...</entry>
-    <entry lang="de" key="IDC_SELECT_PKCS11_MODULE">&amp;Bibliothek auswählen ...</entry>
+    <entry lang="de" key="IDC_SELECT_PKCS11_MODULE">&amp;Bibliothek wählen ...</entry>
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply Translations/Language.de.xml:181]: Unfortunately, the space is missing.

auswählen

wählen

File: Translations/Language.de.xml [hunk 2]
```diff
@@ -44,9 +44,9 @@
     <entry lang="de" key="IDC_HIDVOL_WIZ_MODE_DIRECT">Direkter Modus</entry>
     <entry lang="de" key="IDC_HIDVOL_WIZ_MODE_FULL">Kompletter Modus</entry>
     <entry lang="de" key="IDC_KB">&amp;KB</entry>
-    <entry lang="de" key="IDC_KEYFILES_ENABLE">Schlüsselda&amp;tei verwenden</entry>
-    <entry lang="de" key="IDC_KEYFILES_TRY_EMPTY_PASSWORD">Versuche zuerst ohne Passwort einzubinden</entry>
-    <entry lang="de" key="IDC_KEYFILES_RANDOM_SIZE">Zufällige Größe (64 &lt;-&gt; 1048576)</entry>
+    <entry lang="de" key="IDC_KEYFILES_ENABLE">Schlüsselda&amp;t. verwenden</entry>
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply Translations/Language.de.xml:47]: Unfortunately, the space is missing for "Schlüsseldateien verwenden".

dat1

dat2

File: Translations/Language.de.xml [hunk 3]
```diff
@@ -44,9 +44,9 @@
...
+    <entry lang="de" key="IDC_KEYFILES_TRY_EMPTY_PASSWORD">Einbinden zuerst ohne Passwort versuchen</entry>
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply Translations/Language.de.xml:48]: The verb "versuchen" should be placed at the end just like "anzeigen" for "Passwort anzeigen", for example.

File: Translations/Language.de.xml [hunk 4]
```diff
@@ -213,35 +213,35 @@
...
-    <entry lang="de" key="IDM_DONATE">Jetzt spenden...</entry>
-    <entry lang="de" key="IDM_ENCRYPT_SYSTEM_DEVICE">System-Partition/Laufwerk verschlüsseln ...</entry>
+    <entry lang="de" key="IDM_DONATE">Jetzt spenden ...</entry>
+    <entry lang="de" key="IDM_ENCRYPT_SYSTEM_DEVICE">Systempartition/Systemlaufwerk verschlüsseln ...</entry>
     <entry lang="de" key="IDM_FAQ">Häufig gestellte &amp;Fragen (FAQ)</entry>
     <entry lang="de" key="IDM_HELP">&amp;Benutzerhandbuch</entry>
     <entry lang="de" key="IDM_HOMEPAGE">Webseite</entry>
     <entry lang="de" key="IDM_HOTKEY_SETTINGS">&amp;Tastenkombinationen ...</entry>
-    <entry lang="de" key="IDM_KEYFILE_GENERATOR">&amp;Schlüsseldatei(en) generieren</entry>
+    <entry lang="de" key="IDM_KEYFILE_GENERATOR">&amp;Schlüsseldatei(en) generieren ...</entry>
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply Translations/Language.de.xml]: I aggree.

File: Translations/Language.de.xml [hunk 5]
```diff
@@ -99,32 +99,32 @@
     <entry lang="de" key="IDT_SINGLE_BOOT">Diese Option wählen, wenn sich nur ein Betriebssystem auf dem Computer befindet (auch bei mehreren Benutzern).</entry>
     <entry lang="de" key="IDT_SPEED">Geschw.</entry>
     <entry lang="de" key="IDT_STATUS">Status</entry>
-    <entry lang="de" key="IDT_SYSENC_KEYS_GEN_INFO">Die Schlüssel, der Salt-Wert und andere Daten wurden erfolgreich erstellt. Wenn Sie neue Schlüssel erstellen möchten, klicken Sie auf „Zurück“ und dann auf „Weiter“. Andernfalls klicken Sie auf „Weiter“.</entry>
-    <entry lang="de" key="IDT_SYS_DEVICE">Partition/Laufwerk verschlüsseln auf dem Windows installiert ist. Jeder, der Zugang zum System erlangen möchte, muss das korrekte Passwort bei jedem Start von Windows eingeben. Eine Alternative dazu ist das Erstellen eines versteckten Systems.</entry>
+    <entry lang="de" key="IDT_SYSENC_KEYS_GEN_INFO">Die Schlüssel, der Salt-Wert und andere Daten wurden erfolgreich erstellt. Wenn Sie neue Schlüssel erstellen möchten, klicken Sie auf „Zurück“ und dann auf „Weiter“. Anderenfalls klicken Sie auf „Weiter“.</entry>
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply Translations/Language.de.xml]: I agree

**Comment by idrassi** [comment]
Thank you @Mr-Update. It seems you have addressed all @andreas-becker comments.
I will merge your PR.

**Pull request merged**